utils/tests/field_helpers.rs

```diff
@@ -125,7 +128,7 @@ fn field_big() {
     let field_zero = BaseField::from(0u32);
 
     assert_eq!(
-        BigUint::from_bytes_be(&field_zero.into_repr().to_bytes_be()),
+        BigUint::from_bytes_be(&field_zero.0.to_bytes_be()),
```

Comment:
Can we use a method on it instead of getting the field?

Reply:
Not only we can but we should, since now that I'm looking at it (it was some code written before me) the serialization is not correct. It should be `PrimeField::into_bigint()` instead of `.0`, and these two calls give different bigints (surprisingly!). I'll add a fix here + hotfix on `develop` + regression test.